src/__tests__/ProductView/QueryInterface.js

```diff
@@ -56,6 +56,21 @@ test('Handling save query', () => {
 });
 
```

Comment:
Easier said than done. Do you know how I can get the uncovered lines for branch coverage?